src/Microsoft.Health.Fhir.TemplateManagement/Microsoft.Health.Fhir.TemplateManagement.csproj

```diff
@@ -5,9 +5,28 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <IsPackable>true</IsPackable>
     <NuspecFile>Microsoft.Health.Fhir.Liquid.Converter.nuspec</NuspecFile>
-    <OrasWinUrl>https://github.com/deislabs/oras/releases/download/v0.8.1/oras_0.8.1_windows_amd64.tar.gz</OrasWinUrl>
+    <OrasVersion>0.8.1</OrasVersion>
   </PropertyGroup>
-  
+
+  <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
+    <OsEnv>windows</OsEnv>
+    <OrasUrl>https://github.com/deislabs/oras/releases/download/v0.8.1/oras_0.8.1_windows_amd64.tar.gz</OrasUrl>
```

Comment:
These URLs should use the `OrasVersion` property to keep it dry.

Reply:
done!